lib/uvm/vm-execute.js

```diff
@@ -61,7 +62,8 @@
         lib = requires[i];
         // Play around with the require cache
         try {
-            libraries[lib.object] = __require(lib.require);
+            required = __require(lib.require);
+            libraries[lib.object] = lib.require === 'lodash3' ? required.noConflict() : required;
```

Comment:
I agree with Kunal here. Since we do not have pre and post processors, it is only way that we create debt now and later recover with our browserify changes. Possibly add this as a todo note here before merging.